doc/ip/spi.md

```diff
@@ -12,15 +12,17 @@ In Sonata, there are multiple uses for SPI:
 - Arduino shield
 - mikroBUS click
 
-The offset for each of the blocks is shown below, with each additional block having a `0x1000` offset from the previous.
-
-| SPI Instance          | Offset (from SPI base) |
-|-----------------------|------------------------|
-| LCD Screen            | 0x0000                 |
-| Ethernet MAC          | 0x1000                 |
-| SPI0                  | 0x2000                 |
-| SPI1                  | 0x3000                 |
-| SPI2                  | 0x4000                 |
+The offset for each of the blocks is shown below, with each additional block having a 0x1000 offset from the previous.
+The offsets in the table below are from the SPI base address, which is 0x8030_0000.
+The table also shows the connections of the chip select (CS) wires, some of which are used for block specific functions like reset and data/control signals.
+
+| Name         | Offset | Blocks                    | CS0          | CS1          | CS2          | CS3      |
+|--------------|--------|---------------------------|--------------|--------------|--------------|----------|
+| LCD Screen   | 0x0000 | *Only* LCD                | CS           | Data/control | Reset        |          |
+| Ethernet MAC | 0x1000 | *Only* Ethernet           | CS           | Reset        |              |          |
```

Comment:
For Ethernet/LCD may be worth an explanatory note that we're effectively using the CS lines as special purpose IO rather than chip-select.

Reply:
Thanks added a more explicit sentence on this.